src/Pokemon.java

```diff
@@ -0,0 +1,38 @@
+public abstract class Pokemon {
+    private String name;
+    private int level;
+    private int hp;
+
+    public Pokemon(String name, int level, int hp) {
+        this.name = name;
+        this.level = level;
+        this.hp = hp;
+    }
+
+    abstract void fastAttack();
```

Comment:
ik zou hier van maken:
`public void fastAttack(string attack){
  sout(this.name + "fast attacks with" + attack )
}
`